src/Services/PacketRevisionsService.php

```diff
@@ -55,4 +56,28 @@ public function oneBySlateIdAndRevisionId(string $slateId, string $packetId, str
 
         return PacketRevision::createFromOne($content);
     }
+
+    /**
+     * @param string $slateId
+     * @param string $packetId
+     * @param string $revisionId
+     * @param Update $revisionUpdate
+     * @return PacketRevision
+     */
+    public function update(
+        string $slateId,
+        string $packetId,
+        string $revisionId,
+        Update $revisionUpdate
+    ): PacketRevision {
+        $url = $this->resolveEndpoint('flows/' . $slateId . '/packets/' . $packetId . '/revisions/' . $revisionId);
```

Comment:
```suggestion
        $url = $this->resolveEndpoint('flows/' . $flowId . '/packets/' . $packetId . '/revisions/' . $revisionId);
```
хоч я б заодно відмовився від конкатенації через крапку на користь рядка в подвійних кавичках